docsrc/markdown/datafeed.md

```diff
@@ -113,4 +120,56 @@ datafeed:
     initialIntervalMillis: 500
     multiplier: 2
     maxIntervalMillis: 300000
-```
\ No newline at end of file
+```
+
+## Best practices
```

Comment:
I wonder how we should avoid duplicating this in the future?
I guess we should put this kind of documentation only on the developers website? (at least it could be shared but less open to contribution...)